spec/jobs/searchgov_url_fetcher_job_spec.rb

```diff
@@ -15,6 +15,11 @@
         to raise_error(ArgumentError)
     end
 
+    it 'must have a named argument' do
+      expect { SearchgovUrlFetcherJob.perform_now searchgov_url }.
+        to raise_error(ArgumentError)
```

Comment:
The more specific you can be in your specs, the better. Here, you can specify the error message as well as the error class:
`expect....to raise_error(ArgumentError, 'missing keyword: searchgov_url')`